modules/noise/noise.cpp

```diff
@@ -54,6 +54,7 @@ Vector<Ref<Image>> Noise::_get_seamless_image(int p_width, int p_height, int p_d
 
 Ref<Image> Noise::get_seamless_image(int p_width, int p_height, bool p_invert, bool p_in_3d_space, real_t p_blend_skirt, bool p_normalize) const {
 	Vector<Ref<Image>> images = _get_seamless_image(p_width, p_height, 1, p_invert, p_in_3d_space, p_blend_skirt, p_normalize);
+	ERR_FAIL_COND_V(images.size() == 0, nullptr);
```

Comment:
Given that it's already printing an error in `_get_seamless_image`, this could just return `Ref<Image>()` without printing an error.

Reply:
Thanks! I sightly prefer to do the input validation earlier, I will update it soon.